bin/crew

```diff
@@ -828,6 +831,18 @@ def strip_find_files (find_cmd, strip_option = "")
   system "#{find_cmd} | xargs -r sh -c 'for i in \"$0\" \"$@\"; do case \"$(head -c 4 $i)\" in ?ELF|\!?ar) echo \"$i\";; esac ; done' | xargs -r llvm-strip #{strip_option}"
 end
 
+def strip_dir (dir)
+  Dir.chdir dir do
+    # Strip libraries with -S
+    puts "Stripping libraries..."
+    strip_find_files "find . -type f  \\( -name 'lib*.a' -o -name 'lib*.so*' \\) -print", "-S"
```

Comment:
Why is the `-o` in the middle of `-name`s?

Reply:
Either lib*.a or lib*.so* ...

Reply:
combines two lines into one.

Reply:
Gotcha, nice!